samples/azure/performance-tuning-asb/sample.md

```diff
@@ -0,0 +1,133 @@
+---
+title: ASB Performance Tuning
+summary: Samples that show the impact of different settings and api usage patterns on the performance of the ASB transport.
+component: ASB
+reviewed: 2016-09-06
+related:
+ - nservicebus/azure-service-bus
+ - samples/azure/azure-service-bus
+---
+
+## Prerequisites
+
+include: asb-connectionstring 
+
+Environment variables named `AzureServiceBus.ConnectionString1` and `AzureServiceBus.ConnectionString2` with a different connection string to an Azure Service Bus namespace each.
+
+include: asb-transport
+
+
+## Code walk-through
+
+This sample has three endpoints
+
+* `Sender`
+* `Receiver`
+* `SenderReceiver`
+
+## Sender
+
+`Sender` sends `SomeMessage` to `Receiver`.
+
+snippet: SomeMessage
+
+The sender sends a large amount of messages to the receiver and times how long it takes to do so. The resulting time will be divided by the number of messages sent in order to compute the average throughput. 
+
+There are two variations of `Sender` to show different (good and bad) ways of sending messages, outside the scope of a receive operation, and explains what the impact is on throughput.
```

Comment:
> to show different (good and bad) ways of sending messages, outside the scope of a receive operation, and explains what the impact is on throughput.

to show different ways of sending messages and illustrate what impact they have on performance.